ocs_ci/utility/rhv.py

```diff
@@ -1,7 +1,38 @@
 import logging
+import os
+import yaml
+
+from ocs_ci.ocs import constants
 
 logger = logging.getLogger(__name__)
 
+# default location of files with necessary ovirt cluster details
+OVIRT_CONFIG_FILEPATH = os.path.expanduser("~/.ovirt/ovirt-config.yaml")
+
+
+def load_ovirt_config():
```

Comment:
Is this a work in progress? This looks identical to [load_auth_config](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L1984-L2005).

Reply:
Removing this implementation and accessing via config